Documentation/HowToGuides/Basics/UsingTheTrackedAliasWithTheUnityXRCameraRig/README.md

```diff
@@ -0,0 +1,55 @@
+&gt; [Home](../../../../README.md) &gt; [How-to Guides](../../README.md) &gt; [Basics](../README.md)
+
+# Using The TrackedAlias With The UnityXRCameraRig
+
+  > * Level: Beginner
+  >
+  > * Reading Time: 2 minutes
+  >
+  > * Checked with: Unity 2018.1
+
+## Introduction
+
+The TrackedAlias prefab provides a way of attaching elements to camera rigs without specifically tying them to the underlying SDK specific camera rigs.
+
+This is useful if you want to support multiple SDK camera rigs in your scene but don't want to have to duplicate the same child GameObjects to all the relevant camera rigs.
```

Comment:
```suggestion
This is useful if you want to support multiple SDK camera rigs in your scene but don't want to have to duplicate the same child GameObjects for all the relevant camera rigs.
```

Reply:
I'm still not sure `for` is correct here.

Not using a TrackedAlias would mean you have to duplicate the child GameObjects inside each of the SDK CameraRigs, so you're duplicating to (or `into`) the relevant camera rigs.

duplicating for (to me) implies that there is one per camera rig but not necessarily inside the camera rig.

Maybe `into` is a better word?